config.json

```diff
@@ -44,7 +44,8 @@
   ],
   "ignored": [
     "_test",
-    "docs"
+    "docs",
+    "img"
```

Comment:
Ah. I did the same PR for another language yesterday, before that PR to configlet was merged. Do you want me to remove the change?